services/common/rsufwdsnmpset.py

```diff
@@ -502,43 +508,50 @@ def config_init(
             )
     elif snmp_version == "1218":
         # Based on message type, choose the right port
-        # rsu_ip, snmp_creds, dest_ip, udp_port, rsu_index, psid, tx
+        # rsu_ip, snmp_creds, dest_ip, udp_port, rsu_index, psid, security, tx
+        logging.info(f"Security mode: {security}")
```

Comment:
Question: Are these intended to be logging.info? Or are these debug logging statements?

Reply:
Aw these are leftovers from testing. I have removed them. The existing logs in this script are intended to be info based but you could argue either case since we don't really intend running these in INFO mode regardless.